app/services/analytics_events.rb

```diff
@@ -2952,6 +2952,37 @@ def idv_selfie_image_added(
   end
   # rubocop:enable Naming/VariableName,Naming/MethodParameterName
 
+  # rubocop:disable Naming/VariableName,Naming/MethodParameterName,
```

Comment:
For new analytics events, I'd wonder if it's worth trying to address some of what's being identified in these linters, i.e. naming the event properties with snake case. I suspect it might be made difficult by multiple events sharing the same common payload, but it seems like we will want to address that at some point anyways.